packages/amplify_auth_cognito/test/amplify_auth_cognito_getCurrentUser_test.dart

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
+import 'package:amplify_core/amplify_core.dart';
+
+void main() {
+  const MethodChannel authChannel = MethodChannel('com.amazonaws.amplify/auth_cognito');
+  const MethodChannel coreChannel = MethodChannel('com.amazonaws.amplify/core');
+
+  Amplify amplify = new Amplify();
+  AmplifyAuthCognito auth = AmplifyAuthCognito();
+
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  int testCode = 0;
+
+  setUp(() {
+    authChannel.setMockMethodCallHandler((MethodCall methodCall) async {
+      switch(testCode) {
+        case 1:
+          return {
+            "username": "testUser",
+            "userSub": "testSub"
+          };
+          case 2:
+            return throw PlatformException(code: "AMPLIFY_EXCEPTION", message: "AMPLIFY_GET_CURRENT_USER_FAILED", details: {} );
```

Comment:
Do you need to specify `details: {}`? In the definition of the `PlatformExcetpion`, can `{}` be a default value assigned to the optional `details` argument?

Reply:
Yeah, this can just be null: https://api.flutter.dev/flutter/services/PlatformException-class.html